<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsDialog</class>
 <widget class="QDialog" name="CSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="m_pCheckFileOpenUI">
     <property name="text">
      <string>Custom file open UI</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_pCheckFileSaveUI">
     <property name="text">
      <string>Custom file save UI</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="m_pFontUI"/>
   </item>
   <item>
    <widget class="QComboBox" name="m_pFontLog"/>
   </item>
   <item>
    <widget class="QWidget" name="m_pWidgetSpacingDesktop" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_pButtonsDialog">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_pWidgetSpacingMobile" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pButtonsDialog</sender>
   <signal>accepted()</signal>
   <receiver>CSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pButtonsDialog</sender>
   <signal>rejected()</signal>
   <receiver>CSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
